### DISCUSSION TITLE
Default value for TxFinality Depth

### DISCUSSION THREAD

**Discussion by manoranjith** [hyperledger-labs/go-perun#273] [Ideas]
@matthiasgeihs @ggwpez Do you think we could add a `DefaultTxFinalityDepth` constant to the `ethereum/channel` package ?

**Comment by matthiasgeihs** [comment]
We could do that. Which value would you give it?

**Reply by manoranjith** [reply]
Since, we don't yet run our node on the mainnet, I think the same value `3` which we use in tests should work.
